README.md

```diff
@@ -142,46 +152,60 @@ There is no strict limit over what you can do with a `Matrix`,
 
 ### Zoom
 
-#### Transformations
+#### Transformation
+
+The transformation defines the engine **resting position**. It is a keyframe that is reached at
+certain points, like at start-up or when explicitly requested through `setContentSize` or `setContainerSize`.
+
+The keyframe is defined by two elements: 
 
-When the engine becomes aware of the content size, it will apply a base transformation to the content
-that can be controlled through `setTransformation(int, int)` or `app:transformation` and `app:transformationGravity`.
-By default it is applied only once, and defines the starting viewport over our content.
+- a `transformation` value (modifies zoom in a certain way)
+- a `transformationGravity` value (modifies pan in a certain way)
+
+which can be controlled through `setTransformation(int, int)` or `app:transformation` and `app:transformationGravity`.
 
 |Transformation|Description|
 |--------------|-----------|
 |`centerInside`|The content is scaled down or up so that it fits completely inside the view bounds.|
 |`centerCrop`|The content is scaled down or up so that its smaller side fits exactly inside the view bounds. The larger side will be cropped.|
 |`none`|No transformation is applied.|
 
-The engine applies the given transformation, and any minZoom and maxZoom constraints.
-
-If, after this process, the content is bigger than the container, the engine will also apply a
-translation according to the given transformation gravity.
+After transformation is applied, the transformation gravity will reposition the content with
+the specified value. Supported values are most of the Android `Gravity` flags, plus `TRANSFORMATION_GRAVITY_AUTO`.
 
 |Transformation Gravity|Description|
 |----------------------|-----------|
-|`top`|If the content is taller than the view, translate it so that we see the top part.|
-|`bottom`|If the content is taller than the view, translate it so that we see the bottom part.|
-|`left`|If the content is wider than the view, translate it so that we see the left part.|
-|`right`|If the content is wider than the view, translate it so that we see the right part.|
+|`top`, ...|The content is panned so that its *top* side matches teh container *top* side. Same for other values.|
+|`auto` (default)|The transformation gravity is taken from the engine [alignment](#alignment), defaults to `center` on both axes.|
 
-If, after this process, the content is smaller than the container, note that the current
-[Smaller Policy](#smaller-policy) applies.
+**Note: after transformation and gravity are applied, the engine will apply - as always - all the active constraints,
+including minZoom, maxZoom, alignment. This means that the final position might be slightly (or completely) different.**
 
-Note: you can always trigger a new transformation to be applied by using the `setContentSize` or `setContainerSize` APIs.
+For example, when `maxZoom == 1`, is forced to not be any larger than the container. This means that
```

Comment:
This sentence doesn't seem to make sense. Did you mean:

> This can happen for example, when you set `maxZoom == 1`, so the content is forced to not be any larger than the container.